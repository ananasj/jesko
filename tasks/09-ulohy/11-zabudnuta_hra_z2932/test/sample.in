3
1 5 2
4 0 9
8 7 3
3
7
2
3