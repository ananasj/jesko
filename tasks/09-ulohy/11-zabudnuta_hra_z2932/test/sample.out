1 5 2
4 7 9
8 3 0
